apps/dashboard/config/api_docs/whois.yml

```
@@ -141,3 +141,212 @@ faq:
+      a 422 validation_failed error.
+
+  - question: Are batch responses ordered?
+    answer: >-
+      Yes. Results are returned in the same order as the domains sent
+      in the request body.
+
+  - question: Is WHOIS data cached?
+    answer: >-
+      No. WHOIS information is fetched live from upstream WHOIS servers
+      on every request.
+
```

YAMLlint (1.38.0)

[warning] too many blank lines (1 > 0)

(empty-lines)